draft-private-access-tokens.md

```diff
@@ -210,20 +210,22 @@ with the following challenge:
 
 ~~~
 struct {
-   opaque origin_name<1..2^16-1>;
-   opaque issuer_name<1..2^16-1>;
-   opaque redemption_nonce[32];
+    uint8_t version;
+    opaque origin_name<1..2^16-1>;
+    opaque issuer_name<1..2^16-1>;
+    opaque redemption_nonce[32];
 } TokenChallenge;
 ~~~
 
-origin_name:
-: Name of the Origin (ORIGIN_NAME).
+The structure fields are defined as follows:
+
+- "version" is a 1-octet integer. This document defines version 1.
```

Comment:
Maybe somewhere we should toss some wording that consumers should not attempt to parse unknown versions, to try avoiding people just ignoring the version, trying to parse everything else and exploding (*cough* TLS *cough*).

Reply:
Good call. I've added text in all of the places that use version to explain how it is verified and must gate parsing/usage.